.github/workflows/changelog.yml

```
@@ -118,10 +121,42 @@ jobs:
             ```
             Last Updated: ${{ steps.timestamp.outputs.timestamp }}
 
+            You can download this release header image 🌌 [here](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+
             To re-trigger:
             - Via UI, run [Generate Changelog workflow](https://github.com/${{ github.repository }}/actions/workflows/changelog.yml) (`version`): `${{ env.RELEASE_VERSION }}`
             - Via CLI, run [Github CLI](https://cli.github.com/):
             ```bash
             gh workflow run changelog.yml -f version=${{ env.RELEASE_VERSION }} --repo ${{ github.repository }}
             ```
           edit-mode: replace
+
+  generate-changelog-image:
+    needs: parse-release-version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+        with:
+          ref: release/${{ env.RELEASE_VERSION }}
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Bootstrap packages
+        run: npm run bootstrap
+
+      - name: Run changelog-image
+        run: npm run changelog-image
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          if-no-files-found: ignore
```

Probably want to error here if no files are found?
```suggestion
          if-no-files-found: ignore
```